erpnext/setup/doctype/transaction_deletion_log/transaction_deletion_log.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Should be named "Translation Deletion Tool" not log. A log is a static entity. This tool has actions linked.

Reply:
Done.